src/react/src/components/ContributeForm.jsx

```diff
@@ -140,17 +146,33 @@ const ContributeForm = ({
         <div className="control-panel__group">
             {formInputs}
             <div className="form__field">
-                <MaterialButton
-                    onClick={selectFile}
-                    type="button"
-                    variant="outlined"
-                    color="primary"
-                    className="outlined-button"
-                    disableRipple
-                >
-                    Select Facility List File
-                </MaterialButton>
+                <div style={contributeFormStyles.uploaderButtonWrapper}>
+                    <MaterialButton
+                        onClick={selectFile}
+                        type="button"
+                        variant="outlined"
+                        color="primary"
+                        className="outlined-button"
+                        disableRipple
+                    >
+                        Select Facility List File
+                    </MaterialButton>
+                    <FeatureFlag flag={ENABLE_DROMO_UPLOADING}>
```

Comment:
Doesn’t React throw errors locally if the flag is not active?
If it does, I’d recommend passing a stub for the `alternative` argument, like this: <></>